examples/problems/1200_conditional_entropy_variance.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>I think it's good to explicitly set the title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>